onboarding-manager/internal/tinkerbell/bootstrap.go

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+package tinkerbell
+
+import (
+	"github.com/open-edge-platform/infra-onboarding/onboarding-manager/internal/env"
+	"github.com/open-edge-platform/infra-onboarding/onboarding-manager/internal/tinkerbell/templates"
+)
+
+func Bootstrap() error {
+	zlog.Info().Msg("Bootstrapping Tinkerbell state")
+
+	if err := clearAllTemplates(); err != nil {
+		return err
+	}
+
+	if err := createTemplates(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func createTemplates() error {
+	zlog.Info().Msg("Creating pre-defined Tinkerbell templates")
+	for name, tmplData := range templates.TemplatesMap {
+		template := NewTemplate(string(tmplData), name, env.K8sNamespace)
+		if err := CreateTemplate(template); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func clearAllTemplates() error {
+	zlog.Info().Msg("Clearing all existing Tinkerbell templates")
+	allTemplates, err := ListTemplates()
+	if err != nil {
+		return err
+	}
+
+	for _, tmpl := range allTemplates {
+		zlog.Info().Msgf("Deleting Tinkerbell template %q", tmpl.Name)
```

Comment:
This shall delete the template but if the workflows are referring to these one of template, shall continue to function, right?

Reply:
The template must exist in the moment when workflow is created. When it's being creating a template is copied and curated with `hardwareMap`. After that the curated workflow is saved in the Workflow object and template is not needed anymore.

I made a quick experiment locally to confirm the above.